### PR TITLE
Daman p patch 1

### PR DESCRIPTION
**Pull request by daman-p**
I think I'll need to manually add the comments to code again bc it says it cannot automatically merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,21 +30,47 @@ python src/car_racing.py
...
-# Any other details that you think might be useful
-Do we need to cite pieces of work we used/incorporated/found helpful?
+
+**Testing the Model**
+```bash
+python src/car_racing.py --load_weights_from=model_1000 --evaluation_episodes=10 --mode=test 
+```
+
+**Recording the Model**
+```bash
```
Comment (alazareva) [inline README.md]: I think this will break because the code iterates through sub directories. we might want to add another folder with subdirs that have weights or remove this example

File: README.md [hunk 2]
```diff
@@ -30,21 +30,47 @@ python src/car_racing.py
...
+python src/car_racing.py --load_weights_from=model_1000 --evaluation_episodes=3 --mode=record 
+```
+
+### TensorBoard:
+To run TensorBoard, run the following command on a new python terminal. TensorBoard files can be found in /src/tensorboard.
```
Comment (alazareva) [inline README.md]: tensorboard fils are at the root, not under tensorboard

File: README.md [hunk 3]
```diff
@@ -30,21 +30,47 @@ python src/car_racing.py
...
---steps  # Number of total training steps
---evaluation_episodes  # ?????
+--steps  # Number of total training steps.
+--evaluation_episodes  # Total number of test episodes.
 --load_weights_from  # Load weights from a previous run.
 ```
 
-For example, the following arguments will run the code with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
+**Training the Model**  
+For example, the following arguments will train the model with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model.
```
Comment (alazareva) [inline README.md]: where did you get the info about lowered learning rate? i think warmup steps are just initial steps where no the network is only seeing observations and not training.

Comment (daman-p) [reply README.md]: I can't find it again, but I did find [this ](https://books.google.com/books?id=lc6PDwAAQBAJ&pg=PA307&lpg=PA307&dq=keras-rl+why+warm-up&source=bl&ots=AqlGV5SB2-&sig=ACfU3U2LyKFPcE8rYeLiccwbsHomuA7cDA&hl=en&sa=X&ved=2ahUKEwiYxpe79JXoAhUoHzQIHSoaDS8Q6AEwCXoECAkQAQ#v=onepage&q=keras-rl%20why%20warm-up&f=false)that seems to support the idea it doesn't learn during the warm-up steps.

File: README.md (hunk 1 above)
Comment (daman-p) [reply README.md]: The test or record example?

File: README.md (hunk 3 above)
Comment (daman-p) [reply README.md]: Changed it to: "Number of warm-up steps before learning occurs."

File: README.md (hunk 1 above)
Comment (daman-p) [reply README.md]: Removed the recording example. Its just for the presentation and doesn't effect how the code runs, he can still watch live runs

File: README.txt [hunk 4]
```diff
@@ -15,34 +15,53 @@ keras-rl2==1.0.3
...
+--evaluation_episodes  # Total number of test episodes.
 --load_weights_from  # Load weights from a previous run.
 
 
-For example, the following arguments will run the code with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
 
-python src/car_racing.py  --steps=10000 --warmup_steps=500 --load_weights_from=pretrained_model_v1
+**Training the Model**  
+For example, the following arguments will train the model with a limit of 10,000 steps, with 500 warmup steps
+during which the learning rate is lowered, and loading weights from a previous run/model.
```
Comment (alazareva) [inline README.txt]: I'd remove this comment about learning rate

Comment (daman-p) [reply README.txt]: removed it here, and another mention the in car_racing.py

Comment (daman-p) [reply README.txt]: I'm loosing it. Removed it from readme, I already fix the one in car_racing.py
